resources/puppetlabs/lein-ezbake/template/global/ext/debian/ezbake.service.erb

```diff
@@ -44,13 +44,9 @@ UMask=027
 LimitNOFILE=<%= EZBake::Config[:open_file_limit] %>
 <% end -%>
 
-<% unless EZBake::Config[:debian][:pre_start_action].empty? -%>
-PermissionsStartOnly=true
-<% EZBake::Config[:debian][:pre_start_action].each do |action| -%>
```

Comment:
We don't use those hooks for openvox-server/db, so I removed them to make the whole template more readable.